docs/marketplace/dev/glue-api-guides/202106.0/carts-of-registered-users/managing-carts-of-registered-users.md

```diff
@@ -1679,258 +1685,3285 @@ To retrieve all carts, send the request:
 </details>
 
 <details>
-<summary markdown='span'>Response sample with product offers (Marketplace only)</summary>
-
-{% info_block warningBox "Note" %}This option is available only in case you have upgraded your shop to the [Marketplace](/docs/marketplace/user/intro-to-spryker/marketplace-concept.html) provided by Spryker.{% endinfo_block %}
+<summary markdown='span'>Response sample with product offers</summary>
```

Comment:
I would exclude not relative things from the response (E.g. here `concrete-products` can be removed from the request `include` parameter), output is too big.